### PR TITLE
chore(setup): setup next.js environment

### LINKED ISSUE #1: 🔖 - Setup Next env
## ✍🏼 Description of the task

Initialize a basic Next project with the required dependencies.

## ✅ Acceptance criteria

- [x] Install Pnpm
- [x] Install Next
- [x] Configure Next
- [x] Setup a working page with Next

### PR DESCRIPTION
**Pull request by Boyadjie**
🔖 **US :** Closes #1 

---

## 🤔 Why?

To work on the video generator, i have to use the same framework as the one in use on the root project.

## 💻 How?

I've installed Next.js with it's basics dependecies and created some pages to check if everything work properly.

## ✅ How to validate this PR?

You should see :
- in "/" the home page of Next.js
- in "/generate" a single line of text
- in "/generate/hello" a sentence with the words you typed a the end of the path (her you should see "hello")
- in "/test" the text "test page"

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "typescript.tsdk": "node_modules/.pnpm/typescript@4.9.5/node_modules/typescript/lib",
+  "typescript.enablePromptUseWorkspaceTsdk": true
+}
```
Comment (CruuzAzul) [inline .vscode/settings.json:4]: The configuration of your IDE should not be in the project 👉🏼 `.gitignore`

File: app/generate/[slug]/page.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export default function Page({ params }: { params: {slug: string } }) {
+  
+    return (
+      <>
+        <h1>Hi there !</h1>
+        <p>You will generate a video with the title {params.slug}</p>
+      </>
+    );
+  }
```
Comment (CruuzAzul) [inline app/generate/[slug]/page.tsx:9]: You used this page to test the next routing slugs but couldn't we just keep one page knowing that we won't use the others ?

File: app/generate/page.tsx [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export default function Page() {
+  return <h1>This page is to GENRATE videos</h1>;
```
Comment (CruuzAzul) [inline app/generate/page.tsx:2]:
```suggestion
  return <h1>This page is to GENERATE videos</h1>;
```

File: app/generate/head.tsx [hunk 4]
```diff
@@ -0,0 +1,7 @@
+export default function Head() {
+    return (
+      <>
+        <title>Generate !!</title>
+      </>
+    )
+  }
```
Comment (CruuzAzul) [inline app/generate/head.tsx:7]: I think we won't need a head overload, it was cool to test but not mandatory now 👉🏼 🗑️

File: app/page.tsx [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+            <Image
+              src="/vercel.svg"
+              alt="Vercel Logo"
+              className={styles.vercelLogo}
+              width={100}
+              height={24}
+              priority
+            />
+          </a>
+        </div> */}
```
Comment (CruuzAzul) [inline app/page.tsx:31]: No comment in code 🗑️

File: app/test/head.tsx [hunk 6]
```diff
@@ -0,0 +1,7 @@
+export default function Head() {
+    return (
+      <>
+        <title>Test !!</title>
+      </>
+    )
+  }
```
Comment (CruuzAzul) [inline app/test/head.tsx:7]: 🗑️

File: app/test/page.tsx [hunk 7]
```diff
@@ -0,0 +1,3 @@
+export default function Page() {
+  return <h1>Test page</h1>;
+}
```
Comment (CruuzAzul) [inline app/test/page.tsx:3]: Not mandatory now 🗑️

**Review by CruuzAzul — CHANGES_REQUESTED** [review]
Just a few little feedbacks 👀

File: .vscode/settings.json (hunk 1 above)
Comment (Boyadjie) [reply .vscode/settings.json]: Done 👍

File: app/generate/[slug]/page.tsx (hunk 2 above)
Comment (Boyadjie) [reply app/generate/[slug]/page.tsx]: Sure i've removed it and we can keep the page in /generate 👍

File: app/generate/head.tsx (hunk 4 above)
Comment (Boyadjie) [reply app/generate/head.tsx]: Done 👍

File: app/page.tsx (hunk 5 above)
Comment (Boyadjie) [reply app/page.tsx]: Sorry for that, i've removed the comment 👍

File: app/test/head.tsx (hunk 6 above)
Comment (Boyadjie) [reply app/test/head.tsx]: Done 👍

File: app/test/page.tsx (hunk 7 above)
Comment (Boyadjie) [reply app/test/page.tsx]: Done 👍

**Review by Slocaly — APPROVED** [review]

**Review by CruuzAzul — APPROVED** [review]

**Pull request merged**